src/credentials/vue/ServiceItem.vue

```diff
@@ -0,0 +1,286 @@
+<!--
+    This module focuses on the clickable box that represents a specific service/feature
+    on the left side of the screen. It defines the base structure of the component and
+    from there specific service item components can be defined.
+
+    Additionaly, this module provides a state manager to keep track of the state of
+    of the service items.
+ -->
+<template>
+    <button class="service-item-container" :class="classWhenIsSelected" v-on:mousedown="serviceItemClicked">
```

Comment:
```suggestion
    <button class="service-item-container" :class="classWhenIsSelected" @click="serviceItemClicked">
```

Reply:
Thanks for the review!

I found using mousedown feels much more responsive when clicking the items.